client/tx/query.go

```diff
@@ -26,18 +27,18 @@ func QueryTxCmd(cdc *codec.Codec) *cobra.Command {
 		Short: "Matches this txhash over all committed blocks",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// find the key to look up the account
-			hashHexStr := args[0]
```

Comment:
why delete?

Reply:
Because it's only used in one place.

Reply:
But it add meaning! Otherwise, you'd have to guess what it is